src/perception/src/perception/perception_node.cpp

```diff
@@ -162,16 +176,18 @@ void Perception::publish_cones(std::vector<Cluster>* cones) {
 }
 
 void Perception::fov_trimming(pcl::PointCloud<pcl::PointXYZI>::Ptr cloud, double max_distance,
-                              double min_angle, double max_angle) {
+                              double min_angle, double max_angle, double x_discount) {
   pcl::PointCloud<pcl::PointXYZI>::Ptr trimmed_cloud(new pcl::PointCloud<pcl::PointXYZI>);
 
+  double center_x = -x_discount; // assuming (0, 0) as the center
```

Comment:
What is this discount?

Reply:
I will document this also and this will be changed when the main branch is up-to-date with the system testing branch. But this is a way (some more are being explored) to do the pass-trough filter without excluding the two cones that might be aligned with the car horizontally (180º of trimming would be necessary)